function/services/kta/ntlmClient.js

```diff
@@ -19,10 +21,19 @@ const client = {
                 json: jsonRequest
             };
 
-            httpNtlmClient.post(opts, function(err, response) {
+            const startTime = process.hrtime(); // Start timer
+            logger.info(`Sending request to: ${opts.url}`);
+
+            httpNtlmClient.post(opts, async function(err, response) {
```

Comment:
Does this accept an async function as the callback

Reply:
There was nothing in the docs to suggest this would or would not work, but thinking about it generally, you're right that the if the post function is not expecting a promise then it's not going to `await` the async function or catch any errors thrown by it. As such, I've returned it to a synchronous callback and used promise chaining to run the (newly re-named) `logKTAResponseTime` function asynchronously and catch any error it may throw.